src/friendly-iframe-embed.js

```diff
@@ -444,6 +444,16 @@ export class FriendlyIframeEmbed {
       this.visibilityObservable_.fire(this.visible_);
     }
   }
+
+  setCssAnimPlayingState(running) {
+    if (platformFor(this.win).isIos()) {
+      this.win.document.documentElement.classList.toggle(
```

Comment:
1. a. Sounds to me like we could just change the boilerplate? 
     b. The main problem here would be how long the visibility event will take to trigger.
     c. Yes, that use case definitely needs more thought.
2. Yes.
3. Doesn't work, unfortunately. [See example.](http://codepen.io/anon/pen/oWvEKg)